install/kubernetes/helm/istio/values.yaml

```diff
@@ -342,6 +342,10 @@ global:
     # Default setting for service-to-service mtls. Can be set explicitly using
     # destination rules or service annotations.
     enabled: false
+    # If set to true, and a given service does not have a corresponding DestinationRule configured,
+    # or its DestinationRule does not have TLSSettings specified, Istio configures client side
+    # TLS configuration appropriately.
+    autoClientConfig: false
```

Comment:
why use a new name here? I suggest using automaticMtls here. And please set it to true. There is no reason to require people to know that this exists and configure it to true.

Reply:
Change to `automatic`, as it's already under mtls group. The reason I add `client` in the previous name as this mostly affect client side.

Reply:
i thought we wanted to dark launch this first, no?

I agree with @rshriram on the naming, i'd prefer to be autoMTLS or autoMtls.  Having client in the name can trigger more questions than necessary.

Reply:
Changed to `automatic`